CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Adds support for multi-region RTDB.

```

Comment:
Emm, not sure if we should include this in the CHANGE LOG.

After all the backend work is done, this will be the earliest version to support multi-region, but multi-region isn't a thing yet now.